docs/troubleshooting.md

```diff
@@ -7,3 +7,55 @@ If you work on linux and cannot edit some of the project files right after the f
 ## TLS/HTTPS Issues
 
 See more in the [TLS section](tls.md)
+
+## Production issues
+
+### How to properly build fresh images for production use
+
+Remember that, by default, if you run `docker compose up -d`, only the files `compose.yaml` and `compose.override.yaml` will be used.
+See https://docs.docker.com/compose/intro/compose-application-model.
+
+If you need to build images for production environment, you have to use the following command:
+`docker compose -f compose.yaml -f compose.prod.yaml build --no-cache`.
```

Comment:
To maintain a common command format in documents

````suggestion
If you need to build images for production environment, you have to use the following command:

```console
docker compose -f compose.yaml -f compose.prod.yaml build --no-cache
```
````

<br>

Example
https://github.com/dunglas/symfony-docker/blob/65676b92b30cf6dc4bf89dd967ea14ee659fc93f/docs/production.md?plain=1#L82-L89